ansible/playbooks/adhoc/docker_loopback_to_lvm/docker_loopback_to_direct_lvm.yml

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
@joelddiaz go ahead and put this one in openshift-tools, but leave it in openshift-ansible when you create the "clean up moved stuff" PR please.